### ISSUE TITLE
Which training setting used in best model, bin or bin with delta?

### ISSUE THREAD

**Issue by abbyxxn** [hughw19/NOCS_CVPR2019#7]
Thank you for your excellent work! I'm wondering what the COORD_USE_DELTA setting is in taking in your final model, since  COORD_USE_DELTA = False in train.py, and you emphasize that using two loss functions for classification and regression repectively in the paper.

**Comment by hughw19** [comment]
Hi Abby,

The best model uses bin 32.

We experimentally found the combination of two losses can sometime improve NOCS training and yield lower NOCS L2 error. This still depends on network architecture and training. So, this is not something significant.

Also, at test time after running RANSAC for pose fitting, we find the checkpoint using only classification loss is the best. Note that the same L2 error on NOCS doesn’t mean the same quality for RANSAC. Also, the domain gap may degrade the models using two losses more than the ones using only classification loss. 

In short, we didn’t find this combination important for final pose accuracy pn our data. Hope this help you.


Best,
He

**Comment by abbyxxn** [comment]
Thank you very much for your detailed and helpful explanation. I've got a better understanding of this project and the question has been resolved.
